pandas/core/generic.py

```diff
@@ -2444,6 +2444,11 @@ def to_json(
         ``orient='table'`` contains a 'pandas_version' field under 'schema'.
         This stores the version of `pandas` used in the latest revision of the
         schema.
+        When using :class:`ExtensionDtype`-kind columns, the schema fields will
```

Comment:
this is not relevant here

Reply:
This conflicts with this comment https://github.com/pandas-dev/pandas/pull/44722#discussion_r762492051

Reply:
@jmg-duarte For the doc on the method, we don't need this detail, so you can delete this, as that detail is in `doc/source/user_guide/io.rst `